uptane-standard.md

```diff
@@ -176,11 +176,18 @@ In order to be considered “Uptane-compliant,” an implementation MUST follow
 
 ## Automotive Terminology
 
-*Bus*: An internal communications network that interconnects components within a vehicle. A car can have a number of buses that will vary in terms of power, speed and resources.  
+*Bundle*: A set of images released by the repository that is meant to be installed by all ECUs on a vehicle at the same time.
```

Comment:
1. I would say "by multiple ECUs". A bundle does not necessarily contain images for all ECUs.
2. The phrasing is a bit ambiguous IMO. May be understood as if the same set of images was to be installed on every ECU.

Reply:
This was the existing definition for a bundle in the original design specification. Does Ira's proposed change resolve the ambiguity for you?